.github/workflows/run_rc_validation_go_wordcount.yml

```diff
@@ -55,7 +56,10 @@ jobs:
       - name: Fetch Go SDK RC and Tidy Modules
         working-directory: ./sdks/go/examples/wordcount
         run: |
-          go get -d github.com/apache/beam/sdks/v2@${{ github.event.inputs.rc_tag }}
+          mkdir -p beam-test
+          cd beam-test
+          go mod init beam-test
+          go get github.com/apache/beam/sdks/v2/go/pkg/beam@${{ github.event.inputs.rc_tag }}
```

Comment:
I'm confused - what is this job actually doing?

It seems like we're getting the rc go package? But that will not live beyond the duration of this particular job. Should this be part of the prism job so that we use that version of the sdk?

Reply:
I think I fixed all the issues. PTAL